packages/ember-glimmer/tests/integration/helpers/unbound-test.js

```diff
@@ -0,0 +1,545 @@
+
+    this.runTask(() => set(this.context, 'items',  emberA([{ wham: 'bam' }, { wham: 1 }])));
+
+    this.assertText('bam1');
+  }
+
+  ['@htmlbars it should assert unbound cannot be called with multiple arguments']({ throws }) {
+    let willThrow = () => {
+      this.render(`{{unbound foo bar}}`, {
+        foo: 'BORK',
+        bar: 'BLOOP'
+      });
+    };
+
+    throws(willThrow, /unbound helper cannot be called with multiple params or hash params/);
+  }
+
+  ['@htmlbars should render on attributes']() {
+    this.render(`<a href="{{unbound foo}}"></a>`, {
+      foo: 'BORK'
+    });
+
+    this.assertHTML('<a href="BORK"></a>');
+
+    this.runTask(() => this.rerender());
+
+    this.assertHTML('<a href="BORK"></a>');
+
+    this.runTask(() => set(this.context, 'foo', 'OOF'));
+
+    this.assertHTML('<a href="BORK"></a>');
+
+    this.runTask(() => set(this.context, 'foo', 'BORK'));
+
+    this.assertHTML('<a href="BORK"></a>');
+  }
+
+  ['@htmlbars syntax form updates on parent re-render']() {
+    let fooBarInstance;
+    let FooBarComponent = Component.extend({
+      init() {
+        this._super(...arguments);
+        fooBarInstance = this;
+      },
+      foo: 'BORK',
+      barBinding: 'foo'
+    });
+
+    this.registerComponent('foo-bar', {
+      ComponentClass: FooBarComponent,
+      template: `{{unbound foo}} {{unbound bar}}`
+    });
+
+    this.render(`{{foo-bar}}`);
+
+    this.assertText('BORK BORK');
+
+    this.runTask(() => this.rerender());
+
+    this.assertText('BORK BORK');
+
+    this.runTask(() => {
+      set(fooBarInstance, 'foo', 'OOF');
+      this.rerender();
+    });
+
+    this.assertText('OOF OOF');
+
+    this.runTask(() => set(fooBarInstance, 'foo', 'BLIP'));
+
+    this.assertText('OOF OOF');
+
+    this.runTask(() => {
+      set(fooBarInstance, 'foo', 'BORK');
+      this.rerender();
+    });
+
+    this.assertText('BORK BORK');
+  }
+
+  ['@htmlbars sexpr form does not update on parent re-render']() {
+    this.registerHelper('-capitalize', (args) => args[0].toUpperCase());
+
+    let fooBarInstance;
+    let FooBarComponent = Component.extend({
+      init() {
+        this._super(...arguments);
+        fooBarInstance = this;
+      },
+      foo: 'bork'
+    });
+
+    this.registerComponent('foo-bar', {
+      ComponentClass: FooBarComponent,
+      template: `{{-capitalize (unbound foo)}}`
+    });
+
+    this.render(`{{foo-bar}}`);
+
+    this.assertText('BORK');
+
+    this.runTask(() => this.rerender());
+
+    this.assertText('BORK');
+
+    this.runTask(() => {
+      set(fooBarInstance, 'foo', 'oof');
+      this.rerender();
+    });
+
+    this.assertText('BORK');
+
+    this.runTask(() => set(fooBarInstance, 'foo', 'blip'));
+
+    this.assertText('BORK');
+
+    this.runTask(() => {
+      set(fooBarInstance, 'foo', 'bork');
+      this.rerender();
+    });
+
+    this.assertText('BORK');
+  }
+
+  ['@htmlbars sexpr in helper format does not update on parent re-render']() {
```

Comment:
Combination of https://github.com/emberjs/ember.js/blob/bf067f7370c3c3381352d68ef5dfe55d1178c6f7/packages/ember-htmlbars/tests/helpers/unbound_test.js#L230-L247. Some of the existing htmlbars test have incorrect test names.